packages/plugins/plugin-markdown/src/components/DocumentEditor.tsx

```diff
@@ -120,12 +118,11 @@ const DocumentEditor = ({
 
   return (
     <MarkdownEditor
-      id={fullyQualifiedId(doc)}
+      id={id}
       initialValue={initialValue}
       extensions={extensions}
       scrollTo={scrollTo}
-      // TODO(wittjosiah): Ensure selection is within the document.
-      // selection={selection}
+      selection={selection}
```

Comment:
is there a fix for this somewhere? this broke a bunch of my documents because the stored selection was larger than the document length and so I couldn't open them anymore

Reply:
okay I think I see a fix to only allow selection within the document below in `useTextEditor` below 👍

Reply:
yes and i added a specific story also.